src/test/java/org/fisco/bcos/channel/client/TransactionResourceTest.java

```diff
@@ -0,0 +1,149 @@
+                                newTransactionWithProof.getTransactionWithProof().toString());
+                        System.out.println("Test getTransactionByHashWithProof successfully!");
+                    }
+
+                } else if ("getReceipt".equals(args[1])) {
+                    TransactionReceiptWithProof transactionReceiptWithProof =
+                            web3j.getTransactionReceiptByHashWithProof(transactionHash).send();
+                    if (transactionReceiptWithProof == null) {
+                        System.out.println("transactionReceiptWithProof == null");
+                        System.exit(1);
+                    }
+
+                    List<MerkleProofUnit> transactionReceiptProof =
+                            transactionReceiptWithProof
+                                    .getTransactionReceiptWithProof()
+                                    .getReceiptProof();
+                    System.out.println("receiptProof:" + transactionReceiptProof);
+
+                    System.out.println("***********Test getReceiptByHashWithProof************");
+                    TransactionReceiptWithProof newTransactionReceiptWithProof =
+                            transactionResource.getTransactionReceiptWithProof(
+                                    transactionHash, receiptRootHash);
+                    if (newTransactionReceiptWithProof == null) {
+                        System.out.println("Test getReceiptByHashWithProof failed!");
+
+                    } else {
+                        System.out.println(
+                                newTransactionReceiptWithProof
+                                        .getTransactionReceiptWithProof()
+                                        .toString());
+                        System.out.println("Test getReceiptByHashWithProof successfully!");
+                    }
+                } else if ("getAll".equals(args[1])) {
+                    System.out.println(
+                            "***********Test getTransactionAndReceiptWithProof************");
+                    ImmutablePair<TransactionWithProof, TransactionReceiptWithProof> pair =
+                            transactionResource.getTransactionAndReceiptWithProof(
+                                    transactionHash, transactionsRootHash, receiptRootHash);
+                    if (pair == null) {
+                        System.out.println("Test getAll failed!");
+                    } else {
+                        System.out.println("Test getAll successful!");
+                    }
+                } else {
+                    System.out.println("Command not found!");
+                }
+            } else {
+                System.out.println("Please choose follow commands:\n getTrans or getReceipt!");
+            }
+        } catch (Exception e) {
+            System.out.println(e.getLocalizedMessage());
+            System.exit(1);
+        }
+
+        System.exit(0);
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [System.exit() should not be used in J2EE/JEE apps](https://app.codacy.com/app/fisco/web3sdk/pullRequest?prid=4732338)